requirements.txt

```diff
@@ -91,7 +90,7 @@ psycopg2==2.6.2
 ujson==1.35
 sqlparse==0.2.4
 psycogreen==1.0
-django-bulk-update==1.1.10
+django-bulk-update==2.2.0
```

Comment:
The package was renamed from `bulk_update` to `django_bulk_update`: https://github.com/aykut/django-bulk-update/blob/master/CHANGELOG#L13 . Doesn't this break?